src/main/java/nl/scheveschilder/techiteasybackend/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,122 @@
+package nl.scheveschilder.techiteasybackend.controllers;
+
+import nl.scheveschilder.techiteasybackend.exceptions.RecordNotFoundException;
+import nl.scheveschilder.techiteasybackend.models.Television;
+import nl.scheveschilder.techiteasybackend.repositories.TelevisionRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/tv")
+public class TelevisionsController {
+
+    private final TelevisionRepository repos;
+
+    public TelevisionsController(TelevisionRepository repos) {
+        this.repos = repos;
+    }
+
+
+    @GetMapping("/all")
```

Comment:
de /all is hier overbodig, een lege @GetMapping() zegt al eigenlijk haal alles op.